From dc0c3e3040c6197628e76eb084729a8ee582413f Mon Sep 17 00:00:00 2001
From: Chen Jun <chenjun102@huawei.com>
Date: Fri, 26 Feb 2021 08:10:42 +0000
Subject: [PATCH] Revert "[Huawei] RISC-V: Add Microchip PolarFire kconfig
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: dc0c3e3040c6197628e76eb084729a8ee582413f

 option"

hulk inclusion
category: bugfix
bugzilla: 49892
CVE: NA

-------------------------------------------------------------------------

This reverts commit 536f67936ebf0b2fdea99f04b31ceaec09cc6a8f.
Reviewed-by: Mingwang Li <limingwang@huawei.com>

Signed-off-by: Chen Jun <chenjun102@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/riscv/Kconfig.socs | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/riscv/Kconfig.socs b/arch/riscv/Kconfig.socs
index 74d07250ecc5..8a55f6156661 100644
--- a/arch/riscv/Kconfig.socs
+++ b/arch/riscv/Kconfig.socs
@@ -22,13 +22,6 @@ config SOC_VIRT
 	help
 	  This enables support for QEMU Virt Machine.
 
-config SOC_MICROCHIP_POLARFIRE
-	bool "Microchip PolarFire SoCs"
-	select MCHP_CLK_PFSOC
-	select SIFIVE_PLIC
-	help
-	  This enables support for Microchip PolarFire SoC platforms.
-
 config SOC_KENDRYTE
 	bool "Kendryte K210 SoC"
 	depends on !MMU
-- 
2.26.2

